content/providers/provider-terraform/migrate-hcl.md

```diff
@@ -0,0 +1,527 @@
+          toFieldPath: spec.forProvider.region
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.parameters.amiId
+          toFieldPath: spec.forProvider.ami
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.parameters.keyPairName
+          toFieldPath: spec.forProvider.keyName
+
+    - name: ip-test-env
+      base:
+        apiVersion: ec2.aws.upbound.io/v1beta1
+        kind: EIP
+        metadata:
+          labels:
+            testing.upbound.io/example-name: lb
+          name: lb
+        spec:
+          forProvider:
+            instanceSelector:
+              matchControllerRef: true
+      patches:
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.parameters.region
+          toFieldPath: spec.forProvider.region
+```
+
+{{< /expand >}}
+
+Your Composition sets the explicit values you want to deploy based on the
+definition file you created in the preceding step.
+
+## Create a claim
+
+Now that you have a definition and a composition, you can create a claim to. The
+claim is a set of resources to deploy within a single namespace. Creating claims
+is comparable to different Terraform workspaces in that resources within one
+namespace don't impact resources within another.
+
+```yaml
+apiVersion: aws.example.corp/v1alpha1
+kind: Instance
+metadata:
+  name: example
+  namespace: default
+spec:
+  parameters:
+    region: us-east-1
+    amiId: ami-0005e0cfe09cc9050
+    keyPairName: test-keypair
+    publicKey: |
+      ssh-rsa AAAAB3NzaC1...
+```
+
+Copy and paste the contents of your public key (typically `~/.ssh/id_rsa.pub`)
+into the `publicKey` field.
+
+## Authenticate with your cloud provider
+
+The Crossplane AWS provider configuration handles authentication. You must
+create a Kubernetes secret file to authenticate with your AWS account.
+
+Crossplane supports AWS authentication with:
+Authentication Keys
+Web Identity
+Service Accounts
+
+For more information on cloud provider authentication, checkout the Provider
+Azure and Provider GCP authentication documentation.
+
+Create a new Kubernetes secret.
+
+```shell
+kubectl -n upbound-system create secret generic aws-creds --from-file=credentials=aws-credentials
+```
+
+Verify your secret with `kubectl describe secret`.
+
+```shell
+kubectl describe secret aws-creds -n upbound-system
+
+Name:         aws-creds
+Namespace:    upbound-system
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+creds:  114 bytes
+```
+
+## Install the provider
+
+```yaml
+apiVersion: aws.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: aws-provider-ec2
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: aws-creds
+      key: credentials
+```
+
+```shell
+kubectl apply -f aws-provider-config.yaml
+```
+
+## Deploy your claim
+
+Next, apply your Crossplane Composition.
+
+```shell
+kubectl apply -f claim.yaml
+```
+
+## Verify your deployment
```

Comment:
You should add the `kubectl get claim...` output to show when it's ready instead of just waiting forever for SSH